src/frontend/screens/YourTeam/TeamMemberCard.tsx

```diff
@@ -38,7 +32,7 @@ export const TeamMemberCard = ({
 
   return (
     <TouchableOpacity onPress={onPress} style={styles.container}>
-      <DeviceIcon deviceType={deviceType} />
+      <DeviceIcon deviceType={deviceType} size={30} />
```

Comment:
this is becoming a very deeply nested component, which will make refactoring difficult. I think it would be better to just have a prop `{Icon: ReactNode}` and then just pass `<DeviceIcon deviceType={deviceType} size={30} />` into the component. 

eg

```ts
<TeamMemberCard
   Icon={<DeviceIcon deviceType={deviceType} size={30} />}
   {...rest}
/>

```